packages/storefront/packages/nuxt/src/runtime/defineSdkConfig.template

```diff
@@ -57,12 +55,13 @@ export function defineSdkConfig<TConfig>(config: Config<TConfig>) {
       headers: requestHeaders,
     });
 
-    const getCookieHeader = () => useRequestHeaders(["cookie"]);
+    const getRequestHeaders = () => useRequestHeaders();
 
     return config({
       buildModule,
+      moduleFromEndpoints,
       middlewareUrl,
-      getCookieHeader
+      getRequestHeaders
```

Comment:
I'm completely fine with this change, just please remember that it's a breaking change, so we will have to bump the major

Reply:
Thanks for notifying, I'm gonna deprecate it then